boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpuapp.dts

```diff
@@ -122,6 +132,10 @@
 	rx-region = <&cpuppr_cpuapp_ipc_shm>;
 };
 
+ipc0: &cpuapp_cpurad_ipc {
+	status = "okay";
```

Comment:
Why not just set it above, in: https://github.com/zephyrproject-rtos/zephyr/blob/c3d6d6929dfc67df12a4b5f99b0e95f121f3c936/boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpuapp.dts#L122

Reply:
There are few samples which directly points to ipc0, so I wanted to keep compatibility. Is it okay for you?

Reply:
That's okay (though a comment mentioning the reason why `ipc0` is added might be helpful).
But still, you can add the `ipc0` label in line 122 instead of creating another entry here.